### ISSUE TITLE
Images for a newly uploaded show will not fetch

### ISSUE THREAD

**Issue by Celludriel** [jellyfin/jellyfin#5130]
I've added a new show to my jellyfin and it seems the images cannot be fetched anymore.  I'm on a computer that uses NLD BEP internationalization should , and . be an issue on this

 - OS: Windows 10
 - Virtualization: None
 - Clients: Browser
 - Browser: Chrome
 - Jellyfin Version: Jellyfin version: 10.7.0
 - Playback:
 - Installed Plugins: 

 - Reverse Proxy: none
 - Base URL: none
 - Networking: Host
 - Storage: Remote drive on NAS

**To Reproduce**

1. Add new show to a library
2. Manually add metadata for imdb and themoviedb
3. Refresh metadata
4. No images fetched look in logs for error

**Expected behavior**
Shows images would be downloaded and configured for the new show

**Logs**
```
[03:56:40] [ERR] [188] MediaBrowser.Providers.Manager.ProviderManager: TmdbSeriesImageProvider failed in GetImageInfos for type Series at Z:\-= Series =-\-= Varia =-\The Good Doctor
System.FormatException: Input string was not in a correct format.
   at System.Number.ThrowOverflowOrFormatException(ParsingStatus status, TypeCode type)
   at System.Number.ParseInt32(ReadOnlySpan`1 value, NumberStyles styles, NumberFormatInfo info)
   at MediaBrowser.Providers.Plugins.Tmdb.TV.TmdbSeriesImageProvider.GetImages(BaseItem item, CancellationToken cancellationToken)
   at MediaBrowser.Providers.Manager.ProviderManager.GetImages(BaseItem item, IRemoteImageProvider provider, IReadOnlyCollection`1 preferredLanguages, CancellationToken cancellationToken, Nullable`1 type)
```

**Screenshots**
![image](https://user-images.githubusercontent.com/5664766/106345460-99db8a00-62b0-11eb-8c7f-9cd1c19a22e7.png)

**Comment by Celludriel** [comment]
What I can make up from the log is he is trying to parse an int but the id's of the shows are alphanumerical

71712-the-good-doctor
tt6470478

is it trying to parse out the number from the string and is that parsing going wrong ?

**Comment by cvium** [comment]
71712-the-good-doctor is not a valid tmdbid... If you're using NFOs this will be fixed by #5467
